ecf/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.ecf

```diff
@@ -53,6 +54,8 @@ export PDY=20221231
 
 #Define the directories of your TC genesis stats files
 export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/${evs_ver}/${COMPONENT}/${RUN}/${VERIF_CASE}/stats
+#Define the directories of your NOAA/NWS logos
+export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix/logos
```

Comment:
This won't work for compliance w/ standard EVS variables. You must use the following for FIXevs:
`export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix`.
Downstream usage of $FIXevs will need to append `/logos` to the path. You will need to adjust 
 the `scripts/hurricane/plots/exevs_hurricane_*_plots.sh` scripts to define the following variables as shown below:
`WaterMarkLogoFileName=${FIXevs}/logos/noaa.png`
`WaterMarkLogoFileName=${FIXevs}/logos/nws.png`
`export LOGOroot=${FIXevs}/logos`

Please correct the $FIXevs definition in your 3 plots ecf scripts:
`ecf/hurricane/plots/jevs_hurricane_global_*_plots.ecf`